shipshape/cli/shipshape.go

```diff
@@ -308,6 +315,9 @@ func streamsAnalyze(image, absRoot string, analyzerContainers []string, req *rpc
 }
 
 func pull(image string) {
+	if !docker.OutOfDate(image) {
```

Comment:
Pull is already defined in docker.Pull as a utility. This function is only used within the CLI, where I think we want the policy to apply to all images.